js/foundation.reveal.js

```diff
@@ -85,7 +85,7 @@ class Reveal {
   _makeOverlay(id) {
     var $overlay = $('<div></div>')
                     .addClass('reveal-overlay')
-                    .appendTo('body');
+                    .appendTo('this.options.appendTo');
```

Comment:
In fact, I think I need to change this to a review because without this change this PR is not correct.